.github/workflows/test-application.yaml

```diff
@@ -0,0 +1,105 @@
+name: Test application
+
+on:
+    pull_request:
+    push:
+        branches:
+            - master
+            - '[0-9]+.[0-9]+'
+
+jobs:
+    test:
+        name: "Run tests with php ${{ matrix.php-version }} (${{ matrix.database }}, ${{ matrix.phpcr-transport }})"
+        runs-on: ubuntu-18.04
+
+        env:
+            SYMFONY__DATABASE__DRIVER: pdo_mysql
+            SYMFONY__DATABASE__VERSION: 5.7
+            SYMFONY__DATABASE__HOST: 127.0.0.1
+            SYMFONY__DATABASE__PORT: 3306
+            SYMFONY__DATABASE__NAME: sulu_test
+            SYMFONY__DATABASE__USER: root
+            SYMFONY__DATABASE__PASSWORD: root
+
+        strategy:
+            fail-fast: false
+            matrix:
+                include:
+                    - php-version: '7.0'
+                      database: mysql
+                      phpcr-transport: doctrinedbal
+                      # --prefer-lowest would need a fix in sulu/sulu ^1.6
+                      composer-flags: '--prefer-dist --no-interaction --prefer-stable'
+                      php-extensions: 'mysql, imagick'
```

Comment:
would not configure this in the matrix because we dont need to change it for the form bundle 🙂